tests/test_select.py

```diff
@@ -142,38 +140,21 @@ def page():
             screen.find_by_tag('input').send_keys(Keys.ENTER)
             screen.wait(0.5)
         if new_value_mode == 'add':
-            screen.should_contain("value = ['a', 'd', 'd']" if multiple else 'value = d')
-            screen.should_contain("options = {'a': 'A', 'b': 'B', 'c': 'C', 'd': 'd', 'd': 'd'}" if option_dict else
-                                  "options = ['a', 'b', 'c', 'd', 'd']")
+            screen.should_contain("value = ('a', 'd', 'd')" if multiple else 'value = d')
+            screen.should_contain("options = [Option(label='A', value='a'), Option(label='B', value='b'), Option(label='C', value='c'), Option(label='d', value='d'), Option(label='d', value='d')]")
         elif new_value_mode == 'add-unique':
-            screen.should_contain("value = ['a', 'd']" if multiple else 'value = d')
-            screen.should_contain("options = {'a': 'A', 'b': 'B', 'c': 'C', 'd': 'd'}" if option_dict else
-                                  "options = ['a', 'b', 'c', 'd']")
+            screen.should_contain("value = ('a', 'd')" if multiple else 'value = d')
+            screen.should_contain("options = [Option(label='A', value='a'), Option(label='B', value='b'), Option(label='C', value='c'), Option(label='d', value='d')]")
         elif new_value_mode == 'toggle':
-            screen.should_contain("value = ['a']" if multiple else 'value = None')
-            screen.should_contain("options = {'a': 'A', 'b': 'B', 'c': 'C'}" if option_dict else
-                                  "options = ['a', 'b', 'c']")
-
-
-def test_id_generator(screen: Screen):
-    @ui.page('/')
-    def page():
-        options = {'a': 'A', 'b': 'B', 'c': 'C'}
-        select = ui.select(options, value='b', new_value_mode='add', key_generator=lambda _: len(options))
-        ui.label().bind_text_from(select, 'options', lambda v: f'options = {v}')
-
-    screen.open('/')
-    screen.find_by_tag('input').send_keys(Keys.BACKSPACE + 'd')
-    screen.wait(0.5)
-    screen.find_by_tag('input').send_keys(Keys.ENTER)
-    screen.should_contain("options = {'a': 'A', 'b': 'B', 'c': 'C', 3: 'd'}")
```

Comment:
I think it makes sense that I deleted this test but I probably should add tests for `new_value_to_option`.